src/content/docs/codestream/how-use-codestream/performance-monitoring.mdx

```diff
@@ -251,54 +253,68 @@ If the currently selected repository isn't associated with a service on New Reli
+CodeStream will show the average duration and error rate, for the last 30 minutes, in a CodeLens above each method instrumented by the New Relic APM agents. Metrics are calculated for the service currently selected in the Observability section of CodeStream. Expand a different service and the metrics will update accordingly.
 
-Because all requests by the New Relic agent are not collected all the time, it’s possible that low trafficked methods will not see any data. If data is missing for a specific method that you wish to see results for, custom instrumentation can be used to fill any gaps. See guidance for [Java](/docs/apm/agents/java-agent/custom-instrumentation/java-custom-instrumentation), [.NET](/docs/apm/agents/net-agent/custom-instrumentation/introduction-net-custom-instrumentation), [PHP](/docs/apm/agents/php-agent/features/php-custom-instrumentation), [Python](/docs/apm/agents/python-agent/custom-instrumentation/python-custom-instrumentation), [Ruby](/docs/apm/agents/ruby-agent/api-guides/ruby-custom-instrumentation), [Go](/docs/apm/agents/go-agent/instrumentation/instrument-go-transactions) and [Node.js](/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-custom-instrumentation). Also note that for Node.js, code-level metrics are not captured for anonymous functions.
-
-Your code reviews can now also be more data driven by leveraging performance metrics during the review process. Code-level metrics are displayed in the diffs for both pull requests and feedback requests, allowing you to make sure that performance issues get addressed as part of the changes. 
+Code-level metrics also help make code review more data driven, with CodeLenses being displayed in pull request diffs reviewed in CodeStream.
 
 <img
-  title="A diff with golden metrics."
-  alt="A screenshot of a diff with golden metrics."
+  title="A pull request diff with code-level metrics."
+  alt="A screenshot of a pull request diff with code-level metrics."
   src={codestreamCodeLevelDiffs}
 />
 
-Click the CodeLens to see charts visualizing each of the metrics. If the repository is associated with multiple New Relic-monitored entities, you can easily switch between them. This will change the entity upon which the code-level metrics displayed in the editor are based.
+Click the CodeLens to see charts visualizing each of the metrics. If the repository is associated with multiple New Relic-monitored services, you can easily switch between them. This will change the service upon which the code-level metrics displayed in both the charts and the editor are based, and also change the service selection in the Observability section.
```

Comment:
```suggestion
Click the CodeLens to see charts visualizing each of the metrics. If the repository is associated with multiple New Relic-monitored services, you can easily switch between them. This will change the service upon which the code-level metrics displayed in both the charts and the editor are based, and also change the service selection in the **Observability** section.
```